digital-deck-server/services/cardDistService.js

```diff
@@ -0,0 +1,58 @@
+var Player = require("../services/player.js");
+var Table = require("../services/table.js");
+
+module.exports = {
+  /**
+   * Distributes the cards to the players based on the params.
+   *
+   * @param decks The number of decks to be used in the game session.
+   * @param players The number of players in a session.
+   * @param cardsPerPlayer The number of cards to give per player.
+   * @param cardsOnTable The number of cards to put on the table.
+   * @return {json} The json object containing players with player ids and cards assigned to them, table and cards of table and remaining cards on the deck.
+   */
+  distCards: function (decks, players, cardsPerPlayer, cardsOnTable) {
```

Comment:
Is there a reason you're doing this instead of `function distCards(...) {}`?

Reply:
yea, it's syntax issue. It doesn't let me export the function otherwise.

Reply:
huh weird. Ok, that's fine then